applications/nightly_build/test_infogan.py

```diff
@@ -1,3 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+
 ###############################################################################
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for
```

Comment:
The MIT license information is still here, should it be removed as well?

Reply:
Thanks for the catch. Let me update my script for identifying it.